app/worker.php

```diff
@@ -2,280 +2,109 @@
-    return new Func($queue);
-}, ['queue']);
-
-Server::setResource('queueForCertificates', function (Connection $queue) {
-    return new Certificate($queue);
-}, ['queue']);
-
-Server::setResource('queueForMigrations', function (Connection $queue) {
-    return new Migration($queue);
-}, ['queue']);
+        return $dbForConsole->getDocument('projects', $project->getId());
+    });
 
-Server::setResource('logger', function (Registry $register) {
-    return $register->get('logger');
-}, ['register']);
+$abuseRetention
+    ->setName('abuseRetention')
+    ->setCallback(function () {
+        return DateTime::addSeconds(new \DateTime(), -1 * System::getEnv('_APP_MAINTENANCE_RETENTION_ABUSE', 86400));
+    });
 
-Server::setResource('pools', function (Registry $register) {
-    return $register->get('pools');
-}, ['register']);
+$auditRetention
+    ->setName('auditRetention')
+    ->setCallback(function () {
+        return DateTime::addSeconds(new \DateTime(), -1 * System::getEnv('_APP_MAINTENANCE_RETENTION_AUDIT', 1209600));
+    });
 
-Server::setResource('deviceForFunctions', function (Document $project) {
-    return getDevice(APP_STORAGE_FUNCTIONS . '/app-' . $project->getId());
-}, ['project']);
+$executionRetention
+    ->setName('executionRetention')
+    ->setCallback(function () {
+        return DateTime::addSeconds(new \DateTime(), -1 * System::getEnv('_APP_MAINTENANCE_RETENTION_EXECUTION', 1209600));
+    });
 
-Server::setResource('deviceForFiles', function (Document $project) {
-    return getDevice(APP_STORAGE_UPLOADS . '/app-' . $project->getId());
-}, ['project']);
+$queueForUsageDump
+    ->setName('queueForUsageDump')
+    ->inject('queue')
+    ->setCallback(function (Connection $queue) {
+        return new UsageDump($queue);
+    });
 
-Server::setResource('deviceForBuilds', function (Document $project) {
-    return getDevice(APP_STORAGE_BUILDS . '/app-' . $project->getId());
-}, ['project']);
+$deviceForCache
+    ->setName('deviceForCache')
+    ->inject('project')
+    ->setCallback(function (Document $project) {
+        return getDevice(APP_STORAGE_CACHE . '/app-' . $project->getId());
+    });
 
-Server::setResource('deviceForCache', function (Document $project) {
-    return getDevice(APP_STORAGE_CACHE . '/app-' . $project->getId());
-}, ['project']);
+$deviceForLocalFiles
+    ->setName('deviceForLocalFiles')
+    ->inject('project')
+    ->setCallback(function (Document $project) {
+        return new Local(APP_STORAGE_UPLOADS . '/app-' . $project->getId());
+    });
 
+$container->set($project);
+$container->set($register);
+$container->set($dbForProject);
+$container->set($abuseRetention);
+$container->set($auditRetention);
+$container->set($deviceForCache);
+$container->set($queueForUsageDump);
+$container->set($executionRetention);
+$container->set($deviceForLocalFiles);
 
-$pools = $register->get('pools');
 $platform = new Appwrite();
-$args = $platform->getEnv('argv');
+$args = $_SERVER['argv'];
```

Comment:
We created these wrapper methods in the platform library like getEnv inorder to avoid interacting with the SERVER array directly. We should use the getEnv() method here